lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1771,17 +1771,23 @@ def test_upload_non_existing_user(self):
                        )
 
     def test_upload_only_existing_courses(self):
-        self.create_and_enroll_student(username='a_user')
-        self.create_and_enroll_student(username='b_user')
-        existing_team_1 = CourseTeamFactory.create(course_id=self.test_course_1.id, topic_id='topic_1')
-        existing_team_2 = CourseTeamFactory.create(course_id=self.test_course_1.id, topic_id='topic_2')
+        self.create_and_enroll_student(username='a_user', mode=CourseMode.MASTERS)
+        self.create_and_enroll_student(username='b_user', mode=CourseMode.MASTERS)
+        existing_team_1 = CourseTeamFactory.create(course_id=self.test_course_1.id,
+                                                   topic_id='topic_1',
+                                                   organization_protected=True
+                                                   )
+        existing_team_2 = CourseTeamFactory.create(course_id=self.test_course_1.id,
+                                                   topic_id='topic_2',
+                                                   organization_protected=True
+                                                   )
```

Comment:
Nit: put a newline after the opening paren, so things aren't indented so far over, like in this example: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines